.travis.yml

```diff
@@ -9,6 +9,7 @@ install:
   - make venv
 script:
   - make test
+  - test $TRAVIS_PYTHON_VERSION == '3.7' && venv/bin/check-requirements
```

Comment:
This is [failing](https://travis-ci.org/edmundmok/mealpy/builds/527588437) on python 3.6:
```
$ test $TRAVIS_PYTHON_VERSION == '3.7' && venv/bin/check-requirements
The command "test $TRAVIS_PYTHON_VERSION == '3.7' && venv/bin/check-requirements" exited with 1.
```
Maybe `test $TRAVIS_PYTHON_VERSION != '3.7' || venv/bin/check-requirements`?

Reply:
This is just me being bad at shell scripting. I want `if test $TRAVIS_PYTHON_VERSION == '3.7'; venv/bin/check-requirements; fi`. In 3.6, it'll noop as expected, instead of failing on `test`.